code/datums/loadout/loadout_helper.dm

```diff
@@ -56,8 +56,8 @@
 /proc/buy_stack(obj/item/stack/stack_to_buy_type, datum/loadout_seller/seller, mob/living/user, amount)
 	//Hardcode to check the category. Why is this function even here? But it doesn't work, and here I am doing hardcode to make it work because it's hardcoded anyway.
 	var/item_cat = ""
-	if(user.job.title == SQUAD_LEADER)
-		item_cat = CAT_LEDSUP
+	if(user.job.title == SQUAD_LEADER || FIELD_COMMANDER)
+		item_cat = CAT_LEDSUP || CAT_FCSUP
```

Comment:
This isn't going to do what you want it to.

Reply:
Reverted

Reply:
Did you test vending stacks after changing?

Reply:
Yes, works fine when I tested.

Reply:
Tried it myself. Spawned in vendor, bought a stack of sandbags, put it in backpack, saved loadout. Restarted server, vended loadout, no sandbags. Initial purchasing is fine, loadouts aren't.

Reply:
Does it spend the points and not give sandbags?

Reply:
Nevermind, fixed and tested. Works.